src/Sylius/Component/Payment/Model/PaymentMethodInterface.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Sylius\Component\Payment\Model;
 
+use Sylius\Component\Resource\Model\ResourceInterface;
```

Comment:
Ah no, because it extends PaymentMethodTranslationInterface, I don't like that. But let's fix that in a separate PR I think. :)